package.json

```diff
@@ -1,12 +1,14 @@
 {
   "name": "ng2-auto-complete",
-  "version": "0.5.4",
+  "version": "0.5.5",
```

Comment:
This must be 0.6 since it is a major update. It might be a breaking change. If so, please update it to 1.0.0